library/src/skins/default/src/sass/modules/tool/portal/_portal.scss

```diff
@@ -128,6 +128,10 @@
     width: 26vw;
     height: auto;
   }
+
+  .portal-main-container {
+    display: block !important;
+  }
```

Comment:
why do we need to add his?

Reply:
That is a fix for SAK-50358: improved UI of the mobile view for Assignment and Chatroom tools.